confd/templates/opennms.properties.d/activemq.properties.tpl

```diff
@@ -0,0 +1,23 @@
+#
+# DO NOT CHANGE THIS FILE! GENERATED FROM confd
+#
+{{$base := "/activemq" -}}
+{{if exists (print $base "/broker/ttl") -}}
+org.opennms.jms.timeout={{getv (print $base "/broker/ttl")}}
```

Comment:
It would be nice if keys such as this one matched the property name.

ex.
org.opennms.jms.timeout

would have a key of 
/org/opennms/jms/timeout

If we do this it becomes much more intuitive as to what a given properties key will be.

What do you think? Maybe this is impractical to do.

Reply:
Initially, that was my thought, but as there is no convention on the property names, I opted to build a convention for the key paths on the yaml file that confd is going to use as a reference. As I've used a `base` variable on almost all the templates, changing the parent path will be easy in the future. Unfortunately, to have consistency, we need to fix OpenNMS first.